new_client/src/actions/types.js

```diff
@@ -0,0 +1,11 @@
+export const REGISTER_USER = "REGISTER_USER";
```

Comment:
Not exactly required, but I think using Redux Toolkit is definitely better and easier

Reply:
It was decided (within the SYs) to use Redux only, hence used.

Reply:
Ok